### PR TITLE
Mesh 1132/Update weight polynomial

### PR DESCRIPTION
**Pull request by satyamakgec**
Please merge first #490 PR then it may need some changes as well regarding the fixes in the transaction-payment module, Which will change the actual fee computation and need polynomial to update.

### CODE REVIEW

File: pallets/transaction-payment/src/lib.rs [hunk 1]
```diff
@@ -43,7 +43,7 @@ use polymesh_primitives::{traits::IdentityCurrency, Signatory, TransactionError}
 
 use codec::{Decode, Encode};
 use frame_support::{
-    decl_module, decl_storage,
+    debug, decl_module, decl_storage,
```
Comment (Centril) [inline pallets/transaction-payment/src/lib.rs:46]: Unused?

File: pallets/runtime/develop/src/runtime.rs [hunk 2]
```diff
@@ -227,6 +229,20 @@ pub type DealWithFees = SplitTwoWays<
...
+impl WeightToFeePolynomial for WeightToFee {
+    type Balance = Balance;
+
+    fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
+        smallvec![WeightToFeeCoefficient {
+            degree: 1,
+            coeff_frac: Perbill::from_percent(10),
+            coeff_integer: 0u128, // Coefficient is zero
+            negative: false,
+        }]
```
Comment (maxsam4) [inline pallets/runtime/develop/src/runtime.rs:242]: Does this imply 10_000_000 weight -> 1 POLYX?

File: pallets/staking/src/lib.rs [hunk 3]
```diff
@@ -1946,7 +1946,7 @@ decl_module! {
         /// # Arguments
         /// * origin Required origin for adding a potential validator.
         /// * validator Stash AccountId of the validator.
-        #[weight = 50_000]
+        #[weight = 500_000]
```
Comment (maxsam4) [inline pallets/staking/src/lib.rs:1949]: This looks way too low when compared with other weights.

File: pallets/runtime/develop/src/runtime.rs (hunk 2 above)
Comment (satyamakgec) [reply pallets/runtime/develop/src/runtime.rs:242]: Yes. It is

**Comment by maxsam4** [comment]
@satyamakgec I see that you have increased weight to `10_000_000` but this is still very low. This number implies we can do 200,000 TPS. Seems like we need to add 3 more zeroes across the board.

**Comment by satyamakgec** [comment]
@maxsam4  generally this will offset by the multiplier as well anyway I bump up the weight as well

**Comment by cwsatpolymath** [comment]
Rebase to include the CI build scripts

**Pull request merged**